### PR TITLE
Add Hubot Rocket.Chat adapter

### PR DESCRIPTION
**Pull request by eugulixes**

### CODE REVIEW

**Comment by eugulixes** [comment]
@ukhryab, I expanded the description of the `hubot-happy-birthder` script to answer the question.

File: hubot-rocketchat/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
+FROM cusdeb/alpine3.7-node:amd64
+MAINTAINER Rocket.Chat Team <buildmaster@rocket.chat>
```
[the remark this answers is not in the document]
Comment (eugulixes) [reply hubot-rocketchat/Dockerfile]: Thank you for mentioning it.
> Is the information about maintainer correct?

No, it's not correct.

[the remark this answers is not in the document]
Comment (eugulixes) [reply hubot-rocketchat/Dockerfile]: Done.

[the remark this answers is not in the document]
Comment (ukhryab) [reply hubot-rocketchat/Dockerfile]: What about the "MAINTAINER" instruction itself?

[the remark this answers is not in the document]
Comment (eugulixes) [reply hubot-rocketchat/Dockerfile]: It should be done in all the Docker-images in the project. So, I suggest we will do it later.

**Pull request merged**